js/lykke2.js

```diff
@@ -0,0 +1,1174 @@
+        const payload = this.safeValue (response, 'payload');
+        //
+        //     {
+        //         "payload":[
+        //             {
+        //                 "id":"1b367978-7e4f-454b-b870-64040d484443",
+        //                 "timestamp":1644155923357,
+        //                 "lastTradeTimestamp":1644155923357,
+        //                 "status":"Matched",
+        //                 "assetPairId":"BCHEUR",
+        //                 "type":"Market",
+        //                 "side":"Sell",
+        //                 "price":280.569,
+        //                 "volume":0.01,
+        //                 "filledVolume":0.01,
+        //                 "remainingVolume":0.0,
+        //                 "cost":2.80569
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        return this.parseOrders (payload, market, since, limit);
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            // 'side': 'buy',
+            // 'offset': 0,
+            // 'take': 1,
+            // 'to': 0,
+        };
+        let market = undefined;
+        if (limit !== undefined) {
+            request['take'] = limit; // How many maximum items have to be returned, max 1000 default 100.
+        }
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['assetPairId'] = market['id'];
+        }
+        if (since !== undefined) {
+            request['from'] = since;
+        }
+        const response = await this.privateGetTrades (this.extend (request, params));
+        const payload = this.safeValue (response, 'payload');
+        //
+        //     {
+        //         "payload":[
+        //             {
+        //                 "id":"813a3ffa-1c4b-45cb-b13f-1c077ea2748b",
+        //                 "timestamp":1644155923357,
+        //                 "assetPairId":"BCHEUR",
+        //                 "orderId":"1b367978-7e4f-454b-b870-64040d484443",
+        //                 "role":"Taker",
+        //                 "side":"sell",
+        //                 "price":280.569,
+        //                 "baseVolume":0.01,
+        //                 "quoteVolume":2.8056,
+        //                 "baseAssetId":"2a34d6a6-5839-40e5-836f-c1178fa09b89",
+        //                 "quoteAssetId":"EUR",
+        //                 "fee":null
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        return this.parseTrades (payload, market, since, limit);
+    }
+
+    parseBidAsk (bidask, priceKey = 0, amountKey = 1) {
+        const price = this.safeNumber (bidask, priceKey);
+        let amount = this.safeNumber (bidask, amountKey);
+        if (amount < 0) {
```

Comment:
We must use string-based maths with Precise class here.

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

```JavaScript
const price = this.safeString (bidask, priceKey);
const amount = Precise.stringAbs (this.safeString (bidask, amountKey));
return [ this.parseNumber (price), this.parseNumber (amount) ];
```